db/migrate/20240916122719_create_api_v2_form_documents.rb

```diff
@@ -0,0 +1,11 @@
+class CreateApiV2FormDocuments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :api_v2_form_documents do |t|
+      t.references :form, foreign_key: true, comment: "The form this document belongs to"
+      t.text :tag, null: false,  comment: "The tag for the form, for example: 'live' or 'draft'"
```

Comment:
I've called this tag, based on the [ADR](https://github.com/alphagov/forms/blob/main/ADR/ADR034-api-v2-for-forms.md).

I think it could also be called state, but tag seems a bit nicer.

There is no validation on it, other than it must be present. Do we want to restrict the possible values to draft, live and archived or is that something the admin should be in charge of?

Reply:
Yep, that makes sense to me. I think we agreed as a team that keeping validation minimal in the API is good for now (at least while the only clients are our own apps).